earl/src/main/java/com/einmalfel/earl/RdfItem.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * This file is part of Fineswap Android App.
+ * Copyright (C) Fineswap AS. All rights reserved.
+ *
+ * THIS IS PROPRIETARY SOFTWARE AND IS OWNED BY FINESWAP AS.
+ * DO NOT DISTRIBUTE! DO NOT MODIFY! DO NOT COPY!
+ */
+
+package com.einmalfel.earl;
+
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.util.Log;
+
+import org.xmlpull.v1.XmlPullParser;
+import org.xmlpull.v1.XmlPullParserException;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class RdfItem {
```

Comment:
Have you considered using single Rdf namespace class  for both RSS feed & item? I'm asking because RdfItem and RdfFeed are identical